deployment/.env.dev.dist

```diff
@@ -0,0 +1,15 @@
+# nestjs
+TYPEORM_CONNECTION=postgres
+TYPEORM_HOST=postgres
+TYPEORM_PORT=5432
+TYPEORM_USERNAME=db_user
+TYPEORM_PASSWORD=db_password
+TYPEORM_DATABASE=db_name
+TYPEORM_ENTITIES=dist/**/*.entity.js
+TYPEORM_SYNCHRONIZE="true"
+WAIT_HOSTS=postgres:5432
+NODE_ENV=development
+# db
+POSTGRES_USER=db_user
+POSTGRES_PASSWORD=db_password
+POSTGRES_DB=db_name
```

Comment:
- I believe we could do stg more clever by using only 1 variable name for a given data (eg. POSTGRES_USER) and edit the docker-compose.yaml to do the _glue logic_ for us like so `TYPEORM_USERNAME: $POSTGRES_USER`
- Also, I think we should only put in this file variables that the user can edit safely without breaking the app. What I mean by that is that variables such as TYPEORM_ENTITIES, TYPEORM_SYNCHRONIZE or WAIT_HOSTS do not belong here according to me

Reply:
Ok for the first point. And have you got an idea of how to define "unsafe variables" ? It could be set directly in the docker compose file !